spec/spec_helper.rb

```diff
@@ -1,37 +1,13 @@
-# See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
```

Comment:
As in comment above, we may have some conflict with #22 here. Could we move anything that's directly related to rails and/or the ManageIQ host app into a `spec/rails_helper` file?

Reply:
Agreed, no reason it can't live there. The other places we do this are just wrong. I tried separating stuff on core long long ago but it was too much of a pain, so doing this here right now would keep that from happening to this project, too.